app/code/core/Mage/Install/Block/End.php

```diff
@@ -26,7 +27,7 @@ public function getEncryptionKey()
     {
```

Comment:
Should we just delete this method? It's usage in `install/end.phtml` is removed, and I can't imagine anyone has custom install themes. :)

Reply:
absolutely!